src/attributes.jl

```diff
@@ -1856,6 +1856,86 @@ function attribute_value_type(::ConstraintConflictStatus)
     return ConflictParticipationStatusCode
 end
 
+"""
+    UserDefinedFunction(name::Symbol, arity::Int) <: AbstractModelAttribute
+
+Set a user-defined function by the name of `name` with `arity` arguments.
+
+The value to be set is a tuple containg one to three functions, representing the
+zero, first, and second-order derivatives of the function.
+
+For uni-variate functions with `arity == 1`, the functions must have the form:
+
+ * `f(x::T)::T`: returns the evaluation of the function at `x`
+ * `∇f(x::T)::T`: return the first-order deriative of `f` with respect to `x`
+ * `∇²f(x::T)::T`: returns the second-order derviative of `f` with respect to `x`.
+
+For multi-variate functions, the functions must have the form:
```

Comment:
```suggestion
For multi-variate functions, the functions in the tuple must have the form:
```

Reply:
Can we remove the hyphen in "multi-variate" everywhere? I see in e.g. https://jump.dev/MathOptInterface.jl/dev/submodules/Nonlinear/overview/#Multivariate-operators that the hyphen is not used